shell/src/main/java/tachyon/shell/command/AbstractLsCommand.java

```diff
@@ -46,7 +46,7 @@ protected AbstractLsCommand(TachyonConf conf, TachyonFileSystem tfs) {
    */
   protected void ls(TachyonURI path, boolean recursive) throws IOException {
     List<FileInfo> files = listStatusSortedByIncreasingCreationTime(path);
-    String format = "%-10s%-25s%-15s%-15s%-5s%n";
+    String format = "%-10s%-25s%-15s%-15s%-15s%-5s%n";
```

Comment:
Could you extract this as a constant to be used here and in test? Also, it'll be good to have some comments about the components.